integration-test-groups/aws2/aws-secrets-manager/src/main/java/org/apache/camel/quarkus/component/aws/secrets/manager/it/AwsSecretsManagerResource.java

```diff
@@ -114,10 +127,12 @@ public Response post(@PathParam("operation") String operation, @QueryParam("body
         case describeSecret:
             DescribeSecretResponse response = ex.getIn().getBody(DescribeSecretResponse.class);
             result = CollectionHelper.mapOf("sdkHttpSuccessful", response.sdkHttpResponse().isSuccessful(),
-                    "name", response.name());
+                    "name", response.name(), "description", response.description(),
```

Comment:
So, we have description 2 times here ?

Reply:
Not sure how it got duplicated, but fixed.